backend/app/routers/datasets.py

```diff
@@ -358,6 +360,8 @@ async def edit_dataset(
     db: MongoClient = Depends(dependencies.get_db),
     user_id=Depends(get_user),
     es=Depends(dependencies.get_elasticsearchclient),
+    allow: bool = Depends(DatasetAuthorization("editor")),
```

Comment:
in these cases, you can just use Depends(Authorization( as mentioned above.

Reply:
Fixed this.